packages/components/src/__actions__/Button/v3/_docs/Button--api-specification.mdx

```diff
@@ -0,0 +1,160 @@
+
+`isPending` and `pendingLabel` are used to indicate that an interaction is in progress. This will disable the `Button`'s press events while rendering a loading spinner and `pendingLabel`.
+
+<Canvas of={specStories.PendingButton} />
+
+For an icon only `Button`, this can be paired with the `isPendingLabelHidden` prop to visibly hide the `pendingLabel` to avoid layout shifts. It is important that the `pendingLabel` is still given meaningful message as this will be announced to a screen reader.
+
+### Full width Buttons
+
+If a button is statically the full width of a container you can use `isFullWidth` property.
+
+<Canvas of={exampleStories.ButtonFullWidth} />
+
+For resizing on on smaller screens, consider using the `className` to leverage CSS media or container queries, ie: `<Button className="w-full md:w-[initial]">Label</Button>`.
+
+
+### Styling hover, focus and other pseudo-classes
+
+Pseudo-classes, such as `:hover` or `:focus-visible`, are styled via data attributes that reflect React Aria's Button internal state, ie: `isHovered`, `isFocusVisible`.
+
+When adding styles via the `className` prop, we recommended using [React Aria's data attributes](https://react-spectrum.adobe.com/react-aria/Button.html#styling), such as `:[data-hovered]` or `:[data-focus-visible]`, to ensure styles are applied consistently.
+
+```css
+/* From this: */
+.button:hover {
+  background-color: $color-gray-200;
+}
+
+/* To this: */
+.button:[data-hovered] {
+  background-color: $color-gray-200;
+}
+```
+
+For more on styling with React Aria and other ways to extend the `Button`&apos;s styles, we recommend checking out the [docs here](https://react-spectrum.adobe.com/react-aria/Button.html#styling).
+
+
+### Reversed Colors
+
+The `Button` uses the `useReversedColors` hook and `ReversedColors` provider to control the dark and light theme.
+
+<Canvas className="bg-purple-600 mb-24" of={exampleStories.ReversedButton} />
+
+To enable the reversed theme, you will need to wrap the component or application in the `ReversedColors` provider, ie:
+
+```tsx
+import { Button } from "@kaizen/components/v3/actions"
+import { ReversedColors } from "@kaizen/components/v3/utilities"
+// application code
+
+return (
+  <ReversedColors isReversed={true}>
+    <Button {...buttonProps} />
+  </ReversedColors>
+)
+```
+
+## Additional API options
+
+The following table is a collection of additional React Aria and native HTML props that are exposed from the React Aria Button's API. These are not required for the implementation of `Button` but can be used to extend its functionality.
+
+You can learn more about React Aria's Button API and advance configuration options [here](https://react-spectrum.adobe.com/react-aria/Button.html#props).
+
+<Controls of={exampleStories.Playground} className="mb-64" exclude={["children", "className",  "size", "variant", "onPress", "icon", "iconPosition", "isFullWidth", "isDisabled", "isPending", "isPendingLabelHidden",  "pendingLabel"]}  />
```

Comment:
Slightly unsure about this one. The props are so detached from the example it controls. Maybe use `ArgTypes` instead and also link to this section from the other one?

Reply:
Fair point, here they don't really need to be functional controls 👍 Will link back to the top as well